integration_tests/integration_tests.feature

```diff
@@ -92,51 +92,87 @@ Feature: Run integration tests
+  Examples: Minimum Offer Age
+     | min_offer_age |
+     |       0       |
+     |       1       |
+     |       2       |
+     |       3       |
+     |       4       |
+     |       5       |
+
+  Scenario Outline: Grid fees are calculated based on the clearing rate for pay as clear while dispatching top to bottom
      Given we have a scenario named non_compounded_grid_fees
      And d3a is installed
      And d3a uses an two-sided pay-as-clear market
+     And the minimum offer age is <min_offer_age>
      And d3a dispatches events from top to bottom
      When we run the simulation with setup file non_compounded_grid_fees and parameters [24, 60, 60, 0, 1]
-     Then trades on the House 1 market clear with 28.5 cents/kWh
-     Then trades on the Neighborhood 1 market clear with 28.5 cents/kWh
-     Then trades on the Grid market clear with 27.3125 cents/kWh
-     Then trades on the Neighborhood 2 market clear with 24.9375 cents/kWh
-     Then trades on the House 2 market clear with 23.75 cents/kWh
-
-  Scenario: Grid fees are calculated based on the clearing rate for pay as clear while dispatching bottom to top
+     Then trades on the House 1 market clear with 30 cents/kWh
+     Then trades on the Neighborhood 1 market clear with 30 cents/kWh
+     Then trades on the Grid market clear with 28.75 cents/kWh
+     Then trades on the Neighborhood 2 market clear with 26.25 cents/kWh
+     Then trades on the House 2 market clear with 25 cents/kWh
+  Examples: Minimum Offer Age
+     | min_offer_age |
+     |       0       |
+     |       1       |
+     |       2       |
+     |       3       |
+     |       4       |
+     |       5       |
+
+  Scenario Outline: Grid fees are calculated based on the clearing rate for pay as clear while dispatching bottom to top
      Given we have a scenario named non_compounded_grid_fees
      And d3a is installed
      And d3a uses an two-sided pay-as-clear market
+     And the minimum offer age is <min_offer_age>
      And d3a dispatches events from bottom to top
      When we run the simulation with setup file non_compounded_grid_fees and parameters [24, 60, 60, 0, 1]
-     Then trades on House 1 clear with 30 or 24 cents/kWh
-     Then trades on Neighborhood 1 clear with 30 or 24 cents/kWh
-     Then trades on Grid clear with 28.75 or 22.90909090 cents/kWh
-     Then trades on Neighborhood 2 clear with 26.25 or 20.86956521 cents/kWh
-     Then trades on House 2 clear with 25 or 20 cents/kWh
+     Then trades on the House 1 market clear with 30 cents/kWh
+     Then trades on the Neighborhood 1 market clear with 30 cents/kWh
+     Then trades on the Grid market clear with 28.75 cents/kWh
+     Then trades on the Neighborhood 2 market clear with 26.25 cents/kWh
+     Then trades on the House 2 market clear with 25 cents/kWh
+  Examples: Minimum Offer Age
+     | min_offer_age |
+     |       0       |
```

Comment:
Please reduce number of cases in order to save time in CI.

Reply:
Will do, thanks!